pkgs/top-level/all-packages.nix

```diff
@@ -11859,6 +11859,8 @@ with pkgs;
 
   par2cmdline = callPackage ../tools/networking/par2cmdline { };
 
+  par2cmdline-turbo = callPackage ../tools/networking/par2cmdline-turbo { };
+
```

Comment:
New packages are recommended to go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
Moved. Thanks